src/raul/card

```diff
@@ -0,0 +1,67 @@
+public class Main {
```

Comment:
Let's rename, either the class name or the file name so both match. Also, file extension is missing (should be .java) and your `raul` folder with its content should be moved under `src/credit-card-issuer` folder

Reply:
Hello Jose!
 You said that our folders should be into the src/main/java/creditcardissuer.